flocker/apiclient/test/test_client.py

```diff
@@ -798,6 +815,21 @@ def test_logging(self, logger):
         return d
 
     @capture_logging(None)
+    def test_persistent_connections(self, logger):
+        """
+        ``FlockerClient`` can use a pool with persistent connections when
+        specified.
```

Comment:
Here's someone else saying something similar:
 * http://blog.mailgun.com/stress-testing-http-with-twisted-python-and-treq/#comment-1087501883

Also note that `maxPersistentPerHost = 2 ` by default.